ext/nnstreamer/tensor_decoder/tensordec-tensor_region.c

```diff
@@ -0,0 +1,776 @@
+/**
+ * GStreamer / NNStreamer tensor_decoder subplugin, "bounding boxes"
+ * Copyright (C) 2018 Samsung Electronics Co. Ltd.
+ * Copyright (C) 2018 MyungJoo Ham <myungjoo.ham@samsung.com>

```

Comment:
Remove my name and NXP from this.
Use 2023 instead of 2018.
Add your name (you are the original author of this code).